Assets/Scripts/Shell/BulletController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TankBattle.Tank;
+
+namespace TankBattle.Bullet
+{
+    public class BulletController
+    {
+        private  BulletView bulletView;
+        private  BulletModel bulletModel;
+        public readonly TankController sourceTank;
+
+        public BulletController(BulletScriptableObject _bulletScriptableObject, Vector3 _position, Quaternion _rotation, TankController _sourceTank)
+        {
+            bulletModel = new BulletModel(_bulletScriptableObject);
+            bulletView = GameObject.Instantiate<BulletView>(bulletModel.BulletPrefab, _position, _rotation);
+            bulletView.SetController(this);
+            bulletView.GetComponent<Rigidbody>().AddForce(bulletView.transform.forward * 1000);
```

Comment:
ideally should put this inside a function in view instead of accessing RB outside of the view 
controller should have business logic only